src/app/app.component.html

```diff
@@ -1,20 +1,53 @@
 <!--The content below is only a placeholder and can be replaced.-->
 
 <div class="container">
-    <h2>Welcome to {{ title }}, you can:</h2>
-    <button type="button" class="btn" routerLink="/login">Basic</button>
-    <button type="button" class="btn" routerLink="/signup">Sign up</button>
-    <button type="button" class="btn" routerLink="/organization">Generate Certificates</button>
-    <button type="button" class="btn" routerLink="/student">View your certificates</button> 
+    <button type="button" class="btn" style="background-color:transparent; color:white; font-weight: bold" routerLink="/login">Basic</button>
+    <button type="button" class="btn" style="background-color:transparent; color:white; font-weight: bold" routerLink="/signup">Sign up</button>
+    <button type="button" class="btn" style="background-color:transparent; color:white; font-weight: bold" routerLink="/organization">Generate Certificates</button>
+    <button type="button" class="btn" style="background-color:transparent; color:white; font-weight: bold" routerLink="/student">View your certificates</button>
 </div>
+<div class ="title">
+    <h2>Welcome, to {{ title }}, you can:</h2>
+</div>
+
 <br>
 <h5>Router Outlet</h5>
 <router-outlet></router-outlet>
 <!-- Footer -->
+<style>
+.title {
+  margin-left: 220px;
+}
+.footer {
```

Comment:
Please include all the css in concerned component's css file only. Not in the html file